vm-builder/files/inittab

```diff
@@ -7,7 +7,7 @@
 ::respawn:/neonvm/bin/vector -c /neonvm/config/vector.yaml --config-dir /etc/vector --color never
 ::respawn:/neonvm/bin/chronyd -n -f /neonvm/config/chrony.conf -l /var/log/chrony/chrony.log
 ::respawn:/neonvm/bin/sshd -E /var/log/ssh.log -f /neonvm/config/sshd_config
-::respawn:/neonvm/bin/neonvmd --addr=0.0.0.0:25183
+::respawn:env PATH=/neonvm/bin neonvmd --addr=0.0.0.0:25183 --quota-path=/var/db/postgres/compute
```

Comment:
why is that change and should we also use the same approach for all other binaries from /neonvm/bin?

Reply:
It's because neonvm-daemon now calls 'blkid', 'swapon', etc. binaries. By setting PATH, neonvm-daemon can find them.

Reply:
Got it, thank you for the clarification!